### DISCUSSION TITLE
[FR] Add possibilty to use Video entry

### DISCUSSION THREAD

**Discussion by MacVille** [gicamm/homeassistant-comelit#48] [Ideas]
I'm using an Video entry System of Comelit and would like to integrate that to HA.
The System i'm using is the Mini Wi-Fi (https://www.comelitgroup.com/en/systems/video-entry-technology/the-home-indoor-video-entry-phones/mini-wi-fi/)

This is connected to the Comelit App and can be used for opening the Latch and display the Video from the Outdoor part.
(https://www.comelitgroup.com/en/systems/video-entry-technology/outdoors-entry-phones-and-monitors/quadra/)

**Comment by gicamm** [comment]
The integration supports the comelit vedo and comelit simple prog protocols.
I am trying to figure out how it should support the Mini Wi-fi system and its aim.
